Partially revert 8e2554260, it adds a build dependency (setuptools-markdown)
that we don't have yet.

--- setup.py.orig	Sat Dec 12 21:13:54 2015
+++ setup.py	Sat Dec 12 21:15:38 2015
@@ -35,16 +35,20 @@ python_version = StrictVersion('.'.join(str(n) for n i
 if python_version < StrictVersion('3.5'):
     install_requires.append('enum34')
 
+try:
+    long_description = open('PKG-INFO').read()
+except IOError:
+    long_description = None
+
 exec(open('hypatia/__init__.py').read())
 setup(name='hypatia_engine',
       packages=['hypatia'],
       version=__version__,
       description='2D action adventure game engine',
-      setup_requires=['setuptools-markdown'],
       # pygame isn't on pypi
       # TODO: i should maybe also specifiy Pillow<4
       install_requires=install_requires,
-      long_description_markdown_filename='README.md',
+      long_description=long_description,
       author='Lillian Lemmer',
       author_email='lillian.lynn.lemmer@gmail.com',
       url='http://hypatia-engine.github.io/hypatia',
